rust/server-axum/service.j2

```diff
@@ -4,41 +4,22 @@
 {{ m::do_not_modify() }}
 
 use axum::{
-    body::{Body, BoxBody, HttpBody},
-    extract::{FromRequest, RequestParts},
-    handler::{future::IntoServiceFuture, Handler},
-    http::Request,
-    response::{IntoResponse, Response},
-    routing::{self, on_service, MethodFilter},
-    Router, BoxError,
+    routing::{get, post, delete, patch, put, options, MethodRouter},
+    handler::Handler,
+    Router,
 };
-use futures::{future::LocalBoxFuture, Future};
-use std::{
-    convert::Infallible,
-    marker::PhantomData,
-    pin::Pin,
-    task::{Context, Poll},
-};
-use tower_service::Service;
-
 
-macro_rules! all_the_tuples {
-    ($name:ident) => {
-        $name!(T1);
-        $name!(T1, T2);
-        $name!(T1, T2, T3);
-        $name!(T1, T2, T3, T4);
-        $name!(T1, T2, T3, T4, T5);
-	}
-}
 
 {% for tag in tags %}
 /// {{ tag.tag }} router
-pub struct {{ tag.tag | pascalcase }}Router {
```

Comment:
Generated router from openapi spec with generics was forcing developer to:
- define all of the endpoints with correct paths
- use correct parameters(query, headers, path etc) per endpoint
- use correct request bodies
- allowing to use additional extractors (up to 5)

What is more it guided developer how to register an endpoint in a doc block and statically checked if there is no mistake thanks to the trait.

You limited this functionality to only making sure endpoint is registered.

If you don't want to use these functionalities you can always create router manually without using the generated one. But before we merge this MR i have to restore previous version of the Router.

Reply:
The only purpose of the router is to make sure the rust code stay consistent with api spec.